Objects/Sensor.py

```diff
@@ -61,17 +88,45 @@ def __init__(self, monitor_num, box=None):
             
     def grab_data(self):
         img = self.sct.grab(self.box)
-        return np.array(img)
+        return np.array(img)[...,:3].astype(float)
+    
+    def trigger(self):
+        pw_abs = abs(self.grab_data() - self.mask).astype(np.uint8)
+        bool_mask =  (self.mask == self.background).all(axis=2,  keepdims=True).astype(int)
+        bool_mask = np.ones(bool_mask.shape) - bool_mask
+        mean_sum = np.mean(bool_mask * pw_abs)
+        print(mean_sum)
+        return mean_sum < self.treshold
+    
+    
     
 #Example
-s = ImageSensor(1, (720,100,1000,500))
+#(left, top, widht, height)
+gta_mask = cv2.imread("C:/Users/Santiago/Pictures/gta_stars_mask.png")
```

Comment:
If this will be added to the repository as test code, these values should not be hardcoded, they can be added to the `.env` file or passed as CLI parameters